tests/Unit/Evolution/DgSubcell/Test_Tags.cpp

```diff
@@ -58,6 +58,9 @@ void test() {
       subcell_mesh);
   CHECK(db::get<evolution::dg::subcell::Tags::Coordinates<Dim, Frame::Logical>>(
             logical_coords_box) == logical_coordinates(subcell_mesh));
+
+  TestHelpers::db::test_compute_tag<
+      evolution::dg::subcell::Tags::TciStatusCompute<Dim>>("TciStatus");
```

Comment:
Is the test of the corresponding simple tag missing or am I just not seeing it?

Reply:
Huh, apparently that got lost? I'll add it :)

Reply:
I think I goofed when I replied here... IT appears that the simple tag is tested on line 80 (new) below. The simple tag doesn't depend on dimension :)